### DISCUSSION TITLE
checkbox onChange event only return boolean should include event.target

### DISCUSSION THREAD

**Discussion by spencerchang** [heroui-inc/heroui#634] [Q&A] [answered]
checkbox onChange event only return boolean  but checkbox are used for multi choice usually.
Only return boolean how can i know this checkbox's id or attribute?

For example i wanna making a countries select checkbox but i don't which country i checked or not.

Looking for anyone's help.
Thanks.

**Comment by cyanxiao** [comment]
Matching multiple checkboxes to multiple states one by one (or storing them in a list state) helps to identify different checkbox states. In your case it's good to set up a key value structure (`{countryname: true or false}`) for these states.

**Comment by angelilton** [comment, marked as answer]
hey @spencerchang but here is a good example of the documentation
here is what you looking for https://codesandbox.io/s/tfmx3s?file=/App.js&from-sandpack=true
